src/Func.h

```diff
@@ -2009,6 +2019,11 @@ class Func {
      * it. */
     EXPORT Stage update(int idx = 0);
 
+    /** Set the type of memory this Func should be stored in. Controls
```

Comment:
The comment on where allocations go is perhaps not that helpful. It seems like this information is/should be better communicated in the MemoryType enumeration.